h2o-core/src/main/java/water/api/schemas3/ModelParametersSchemaV3.java

```diff
@@ -122,6 +122,9 @@
           help = "Encoding scheme for categorical features")
   public Model.Parameters.CategoricalEncodingScheme categorical_encoding;
 
+  @API(help="Key of H2OTargetEncoderEstimator or H2OTargetEncoderEstimator", direction=API.Direction.INOUT)
```

Comment:
@sebhrusen should we use here TargetEncoderModel or H2OTargetEncoderEstimator naming? Given that generated code will be used in Py it is better to use xxxEstimator naming. wdyth?

Reply:
```suggestion
  @API(help="Key of Target Encoder model", direction=API.Direction.INOUT)
```

Reply:
These API descriptions should not be language specific -- they are used in Java, Pythom & R (and Flow?).  So given that, I'd just call it "Target Encoding model".

Reply:
@ledell you are right, thanks! But looks like this api will go away